### DISCUSSION TITLE
真的好厉害,这么小的安装包

### DISCUSSION THREAD

**Discussion by sanbei101** [HuLaSpark/HuLa#5] [Q&A] [answered]
![image](https://github.com/nongyehong/HuLa-IM-Tauri/assets/95988545/3a23c6c9-1158-44c7-b2ae-7c432e9e7a08)
如果需要正式使用的话需要怎么做呢

**Comment by nongyehong** [comment]
拉取tauri-im-server后端服务，部署后端服务就可以了

**Reply by akwangl** [reply]
但是不支持分布式

**Reply by nongyehong** [reply, marked as answer]
已支持
